server/server.js

```diff
@@ -220,11 +234,22 @@ Server.prototype._handleResourceMessage = function(socket, message, messageType)
     this._storeResource(resource);
     this._persistenceSubscribe(resource.name, socket.id);
     this._updateLimits(socket, message, resource.options);
+    this._stampMessage(socket, message);
```

Comment:
This is related to the same thing. Messages need to be stamped in order t
be sent to Kafka. The stamp provides a way to contextualize the messages.

On Friday, August 7, 2015, Vanchi Koduvayur <notifications@github.com>
wrote:

> In server/server.js
> <https://github.com/zendesk/radar/pull/149#discussion_r36526083>:
>
> > @@ -220,11 +234,22 @@ Server.prototype._handleResourceMessage = function(socket, message, messageType)
> >      this._storeResource(resource);
> >      this._persistenceSubscribe(resource.name, socket.id);
> >      this._updateLimits(socket, message, resource.options);
> > +    this._stampMessage(socket, message);
>
> this stuff is good. But keep off this PR please. Cherry pick this commit
> out of this branch into new one and PR that if you want. But let's keep
> focus on the actual change. I am finding it hard to review all these giant
> PRs.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendesk/radar/pull/149/files#r36526083>.
>


-- 
Lucas Florio
Software Engineer
Zendesk <http://www.zendesk.com>